torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -109,7 +132,9 @@ std::string getModuleTypeName(const Module& module, const std::string& prefix) {
 std::pair<IValue, c10::optional<IValue>> getFunctionTuple(
     const Module& module,
     const Function& func,
-    bool save_mobile_debug_info) {
+    std::unordered_map<at::Tensor, int, tensor_value_hash, tensor_value_equal>
```

Comment:
Why passing by value instead of const ref?

Reply:
Good catch! Update it in latest commit.